loader/include/Geode/ui/General.hpp

```diff
@@ -86,7 +86,7 @@ namespace geode {
     public:
         static ListBorders* create();
 
-        void setSpriteFrames(const char* topAndBottom, const char* sides, float topPadding = 7.5f);
+        void setSpriteFrames(const char* topAndBottom, const char* sides, float horizontalPadding = 7.5f);

```

Comment:
why the rename?

Reply:
look into how it's used
it's used as both top and bottom padding and ends up being used as horizontal padding for the top and bottom sprite